examples/userapi/03_subdomains.ipynb

```diff
@@ -432,15 +432,7 @@
    "cell_type": "code",
```

Comment:
<p>Leftover output in this cell probably doesn't want to be there</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2252/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>